android/src/main/java/com/reactnativestripesdk/StripeSdkModule.kt

```diff
@@ -271,6 +215,7 @@ class StripeSdkModule(private val reactContext: ReactApplicationContext) : React
   }
 
   @ReactMethod
+  @SuppressWarnings("unused")
   fun initialise(params: ReadableMap, promise: Promise) {
     val publishableKey = getValOr(params, "publishableKey", null) as String
     val appInfo = getMapOrNull(params, "appInfo") as ReadableMap
```

Comment:
On line 278: setUrlSchemeOnAndroid, it appears this is the returnUrl on Android.  I wonder if we can deprecate setUrlSchemeOnAndroid and move to setReturnUrlSchemeOnAndroid?

Let's check with @ccen-stripe, but I think we recommend that user's don't set this, but not sure where we indicate that to users.  ([Stripe.kt](https://github.com/stripe/stripe-android/blob/3f632b03d26853077d47ba80ecf6123664793d0c/payments-core/src/main/java/com/stripe/android/Stripe.kt#L251) has some commentary on the effect of the returnUrl, but doesn't say anything about recommending it is not set.)